Sources/DualCameraKit/DualCameraDisplayView.swift

```diff
@@ -42,22 +42,23 @@ public struct DualCameraDisplayView: View {
                     cameraView(for: .back, widthFraction: 0.5)
                     cameraView(for: .front, widthFraction: 0.5)
                 }
-                .ignoresSafeArea(.all)
+//                .ignoresSafeArea(.all)
                 
             case .stackedVertical:
                 VStack(spacing: 0) {
                     cameraView(for: .back, heightFraction: 0.5)
                     cameraView(for: .front, heightFraction: 0.5)
                 }
-                .ignoresSafeArea(.all)
+//                .ignoresSafeArea(.all)
             }
         }
         .task {
-            do {
-                try await controller.startSession()
-            } catch {
-                print("Camera session error: \(error)")
-            }
+            // TODO: fixme - should View here depend on VM? this functionality led to bug in screen due to ctrl.startSession vs. vm.startSession
```

Comment:
- consider this here for this PR. it led to weird bug. potentially the view should depend on the VM - if not make the VM naming specific for DualCamera**Screen**ViewModel